components/content-service/pkg/storage/minio.go

```diff
@@ -526,8 +532,8 @@ func (s *presignedMinIOStorage) BlobObject(name string) (string, error) {
 }
 
 // BackupObject returns a backup's object name that a direct downloader would download
-func (s *presignedMinIOStorage) BackupObject(workspaceID string, name string) string {
-	return minioWorkspaceBackupObjectName(workspaceID, name)
+func (s *presignedMinIOStorage) BackupObject(workspaceID, name string) string {
+	return minioWorkspaceBackupObjectName("", workspaceID, name)
```

Comment:
What is the reason that the username is not set here but in 
```
func (s *presignedMinIOStorage) Bucket(ownerID string) string {
```
above it is?!

Reply:
I think that is because in this case it is `presignedMinIOStorage`

Reply:
So this feature would only work with directMinioAccess, presignedMinIOStorage (that I think is used by prebuilds, log upload, etc), would not work correctly with dedicated bucket. It would still attempt to access bucket the old way. :thinking:

Reply:
I think presignedMinio would not be working correctly with dedicated bucket. So I committed a fix to it.